src/keys.rs

```diff
@@ -437,6 +431,13 @@ impl From<[u8; 11]> for DiversifierIndex {
     }
 }
 
+impl DiversifierIndex {
+    /// Returns the raw bytes of the diversifier index.
+    pub fn to_bytes(&self) -> &[u8; 11] {
+        &self.0
+    }
+}
+
 impl DiversifierKey {
     /// Returns the diversifier at index 0.
     pub fn default_diversifier(&self) -> Diversifier {
```

Comment:
We could just delete this method. The diversifier at index 0 has no particular significance any more.